src/main/resources/db/changelog/changes/v1.0.1/sql/create-list-details-table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE IF EXISTS list_details ALTER COLUMN fql_query TYPE text;
```

Comment:
Could these changes be incorporated into the update-list-details-table.sql file that already exists? Then we would only need the one file in changelog-v1.0.1.xml.

Reply:
But won't it be much more organized if we have alter columns in one and updating the column values in another. Like 
one for updating and one for altering ?